### DISCUSSION TITLE
Helm chart for IdentityServer4.Admin to deploy on k8s

### DISCUSSION THREAD

**Discussion by bravecobra** [skoruba/IdentityServer4.Admin#857] [Show and tell]
I started the basis for a helm chart for Identityserver4.Admin to make it easier to deploy on a k8s cluster.

It is still a work in progress though as there is no TLS support yet, the API service is still missing, only the default namespace will work and you can only use the embedded SQL server instance to store you data. However, I'd like to add all those feature in the future.

Note that you need to cover DNS and Ingress support as explained in the readme.

You can find in my github repo at [https://github.com/bravecobra/identityserver4.admin-helm](https://github.com/bravecobra/identityserver4.admin-helm).

Happy helming!

**Comment by skoruba** [comment]
Hey @bravecobra - this is a great news, I will definitely test it. Thanks! 😊

**Comment by bravecobra** [comment]
I just added TLS support. You can now run it on both http or https. Either provide your own certificates or let them be generated by `cert-manager`. Next is some namespacing and adding support for external database services.

**Reply by bravecobra** [reply]
I added support for MySQL and Postgres as well. You can now also provide your own database if you want to. Namespacing is also fixed. Only thing left to do is adding the API service

**Reply by bravecobra** [reply]
And that last one got added as well. You can optionally add the API if wanted. 
I'm hitting that same problem described in https://github.com/skoruba/IdentityServer4.Admin/issues/847. Lemme know when you provided a new version of the docker image, so I can update the version number to use.
